### PR TITLE
chore: revert SDK version change to tab/function template

### PR DESCRIPTION
**Pull request by blackchoey**
Graph toolkit has peer dependency to teamsfx SDK. Revert the SDK version changes for current release to avoid package install errors. We can bump the SDK version later to support the preview feature.

### CODE REVIEW

**Review by hund030 — APPROVED** [review]

**Review by wenytang-ms — APPROVED** [review]

File: templates/tab/js/default/package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
         "@microsoft/mgt-teamsfx-provider": "2.4.1-next.teamsfx.3511fba",
         "@microsoft/microsoft-graph-client": "^3.0.1",
         "@microsoft/teams-js": "^1.9.0",
-        "@microsoft/teamsfx": "0.7.0-rc.0",
+        "@microsoft/teamsfx": "^0.6.0",
```
Comment (adashen) [inline templates/tab/js/default/package.json:12]: 0.6.1?

File: templates/tab/ts/default/package.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
         "@microsoft/mgt-teamsfx-provider": "2.4.1-next.teamsfx.3511fba",
         "@microsoft/microsoft-graph-client": "^3.0.1",
         "@microsoft/teams-js": "^1.9.0",
-        "@microsoft/teamsfx": "0.7.0-rc.0",
+        "@microsoft/teamsfx": "^0.6.0",
```
Comment (adashen) [inline templates/tab/ts/default/package.json:12]: 0.6.1?

Comment (blackchoey) [reply templates/tab/ts/default/package.json:12]: the previous tab template uses 0.6.0, I didn't change the old value here

File: templates/tab/js/default/package.json (hunk 1 above)
Comment (blackchoey) [reply templates/tab/js/default/package.json:12]: the previous tab template uses 0.6.0, I didn't change the old value here

**Review by SLdragon — APPROVED** [review]

**Review by adashen — APPROVED** [review]

**Pull request merged**